Remotion/Core/Reflection.UnitTests/CodeGeneration/MethodWrapperEmitterTests/SetUpFixture.cs

```diff
@@ -65,17 +58,6 @@ public virtual void OneTimeSetUp ()
     public virtual void OneTimeTearDown ()
     {
       Console.WriteLine("Tearing down MethodWrapperEmitterTests");
-#if NETFRAMEWORK && ENABLE_PEVERIFY
-
-      s_assemblyBuilder.Save(s_moduleBuilder.ScopeName);
-      var path = s_moduleBuilder.FullyQualifiedName;
-
-      s_assemblyBuilder = null;
-      s_moduleBuilder = null;
-
-      PEVerifier.CreateDefault().VerifyPEFile(path);
```

Comment:
please check PEVerify
please check occurances of ENABLE_PEVERIFY

Reply:
Do we want to have no occurrences of this? What is the goal with checking it?

Reply:
Theres a comment about this lower down, I guess we want to remove all usages of it.